Pipfile

```diff
@@ -0,0 +1,50 @@
+[[source]]
+url = "https://pypi.org/simple"
+verify_ssl = true
+name = "pypi"
+
+[requires]
+python_version = "3.5"
```

Comment:
Does this need a >=?

Reply:
Pipenv pins every dependency so does the Python version. If this doesn't fit our requirements, we have to remove the **[requires]** section. For now it is not supported by pipenv to specify a range of Python versions.